### DISCUSSION TITLE
Custom Form/View Actions

### DISCUSSION THREAD

**Discussion by GreatPotato** [filamentphp/filament#4837] [Help]
Hey guys, new to Filament but absolutely loving it.

One thing I cant seem to find in the documentation is anything about custom actions?

For example, say on the demo in the order screen, could you add a button to "Send to Accounting Software" and when its clicked it could integrate with a 3rd party api or similar?

Maybe im missing something...!?

**Comment by danharrin** [comment]
Try this - https://filamentphp.com/docs/2.x/admin/pages/actions

**Reply by GreatPotato** [reply]
But thats only for a custom page right? What about on say a Resource form?

**Reply by danharrin** [reply]
You can use these inside the `getActions()` array of any page, including resource pages.

**Reply by GreatPotato** [reply]
Ok I misunderstood the docs - ok that is amazing. Love it works perfectly!
